packages/@aws-cdk/aws-lambda-nodejs/parcel-bundler/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:13.8.0-alpine3.11
```

Comment:
Add a wee comment explaining what this docker image is

Reply:
Done